### DISCUSSION TITLE
Where is the "Enable Realtime" schema change tracked?

### DISCUSSION THREAD

**Discussion by ChuckJonas** [supabase/supabase#14817] [Questions]
I'm running the supabase local DX and running into a problem where every time I run `supabase db reset` the "Enable Realtime" on my tables is not checked.

I tried running `supabase db diff` to make sure I wasn't missing anything in my migrations, but that doesn't return any differences.

<img width="650" alt="chat___Default_Project___Default_Organization___Supabase" src="https://github.com/supabase/supabase/assets/5217568/5ce054d1-9511-41b8-86c2-e4af5bc9249b">

**Comment by GaryAustin1** [comment]
It can be done with SQL.  See the example code here:  https://supabase.com/docs/guides/realtime/subscribing-to-database-changes#setup

**Reply by ChuckJonas** [reply]
Thanks! Any idea why `supabase db diff` doesn't show these changes?
